.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 sudo: false
 
 python:
-    - 3.6
+    - 3.5
     - 3.7-dev
 
 matrix:
```

Comment:
So something like the following?
```
matrix:
  include:
    - python: 3.5
      env:
        - STYLE=0
        - COVERAGE=0
    - python: 3.6
      env:
        - STYLE=1
        - COVERAGE=0
    - python: 3.6
      env:
        - STYLE=0
        - COVERAGE=1
    - python : 3.7-dev
      env:
        - STYLE=0
        - COVERAGE=0
```

Reply:
I _think_ we could even remove the duplicate `- python: 3.6` so it might look like:

```
matrix:
  include:
    - python: 3.5
      env:
        - STYLE=0
        - COVERAGE=0
    - python: 3.6
      env:
        - STYLE=1
        - COVERAGE=1
    - python : 3.7-dev
      env:
        - STYLE=0
        - COVERAGE=0
```

Reply:
My guess is that that would run STYLE and COVERAGE sequentially, rather than in parallel, which would make the 3.6 test take longer.

Reply:
It's true. I guess I'm less worried about the timing, here, but it's a good point. Let's keep them separate for now then !